frontend/src/components/common/checkbox-list/checkbox-list.tsx

```diff
@@ -32,11 +32,11 @@ const CheckboxList: FC<Props> = ({
 
   const handleChange = (event: React.ChangeEvent<HTMLInputElement>): void => {
     const value = event.target.value;
-    const data = checkedList.includes(value)
+    const list = checkedList.includes(value)
       ? checkedList.filter((item) => item !== value)
       : [...checkedList, value];
```

Comment:
```suggestion
    const list = checkedList.includes(value)
      ? checkedList.filter((item) => item !== value)
      : checkedList.concat(value)
```

Just a style change :slightly_smiling_face: 
If we use the `.smth` above, usage of the `.concat` will look pretty consistent